### PR TITLE
1 setup repo react app

### PR DESCRIPTION
**Pull request by AmirNaghibi**
- create react application 
- containerize app
- setup instructions

issue: #1

### CODE REVIEW

**Comment by AmirNaghibi** [comment]
@engichang1467 feel free to review and leave comment.

File: .gitignore [hunk 1]
```diff
@@ -0,0 +1,23 @@
...
+# production
+/build
+
+# misc
+.DS_Store
+.env.local
+.env.development.local
+.env.test.local
+.env.production.local
+
```
Comment (engichang1467) [inline .gitignore:20]: Can you please add .dist as well?
It's a folder that kept auto generates everytime when I open the project with vscode

**Review by engichang1467 — CHANGES_REQUESTED** [review]
On .gitignore, please add .dist

File: .gitignore (hunk 1 above)
Comment (AmirNaghibi) [reply .gitignore:22]: added `.dist` and `.vscode` to gitignore

**Review by engichang1467 — APPROVED** [review]
Looks good now

**Comment by AmirNaghibi** [comment]
@engichang1467 Great! 🙏🏼

Generally, whoever is the PR creator should merge the PR after approval.

**Pull request merged**